lib/client-options.js

```diff
@@ -10,35 +10,38 @@ const auth = require("./auth");
 
 /**
  * Client options.
- * <p>While the driver provides lots of extensibility points and configurability, few client options are required.</p>
- * <p>Default values for all settings are designed to be suitable for the majority of use cases, you should avoid
- * fine tuning it when not needed.</p>
- * <p>See [Client constructor]{@link Client} documentation for recommended options.</p>
+ *
+ * While the driver provides lots of extensibility points and configurability, few client options are required.
+ *
+ * Default values for all settings are designed to be suitable for the majority of use cases, you should avoid
+ * fine tuning it when not needed.
+ *
+ * See [Client constructor]{@link Client} documentation for recommended options.
+ *
  * @typedef {Object} ClientOptions
  * @property {Array.<string>} contactPoints
  * Array of addresses or host names of the nodes to add as contact points.
- * <p>
- *  Contact points are addresses of Cassandra nodes that the driver uses to discover the cluster topology.
- * </p>
- * <p>
- *  Only one contact point is required (the driver will retrieve the address of the other nodes automatically),
- *  but it is usually a good idea to provide more than one contact point, because if that single contact point is
- *  unavailable, the driver will not be able to initialize correctly.
- * </p>
+ *
+ * Contact points are addresses of Cassandra nodes that the driver uses to discover the cluster topology.
+ *
+ * Only one contact point is required (the driver will retrieve the address of the other nodes automatically),
+ * but it is usually a good idea to provide more than one contact point, because if that single contact point is
+ * unavailable, the driver will not be able to initialize correctly.
+ *
  * @property {String} [localDataCenter] The local data center to use.
- * <p>
- *   If using DCAwareRoundRobinPolicy (default), this option is required and only hosts from this data center are
- *   connected to and used in query plans.
- * </p>
+ *
+ * If using DCAwareRoundRobinPolicy (default), this option is required and only hosts from this data center are
+ * connected to and used in query plans.
+ *
```

Comment:
It's my mistake, but this field is missing TODO. Only ``contactPoints`` is somehow supported

Reply:
Ok, I fixed that.